<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LotDialogClass</class>
 <widget class="QDialog" name="LotDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Lot</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/res.qrc">
    <normaloff>:/pic/field.png</normaloff>:/pic/field.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>District:</string>
        </property>
        <property name="buddy">
         <cstring>lineEditGark</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditGark"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Block:</string>
        </property>
        <property name="buddy">
         <cstring>lineEditFlur</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditFlur"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Lot No.:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxLotNr</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxLotNr">
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Area:</string>
        </property>
        <property name="buddy">
         <cstring>doubleSpinBoxLotSize</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxLotSize">
        <property name="decimals">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Harvest Year:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxYear</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="spinBoxYear">
        <property name="minimum">
         <number>1970</number>
        </property>
        <property name="maximum">
         <number>2100</number>
        </property>
        <property name="value">
         <number>2008</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBoxLotSubNr">
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Lot Sub No.:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxLotSubNr</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>lineEditGark</tabstop>
  <tabstop>lineEditFlur</tabstop>
  <tabstop>spinBoxLotNr</tabstop>
  <tabstop>spinBoxLotSubNr</tabstop>
  <tabstop>doubleSpinBoxLotSize</tabstop>
  <tabstop>spinBoxYear</tabstop>
 </tabstops>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LotDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LotDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
